src/core/glossary.md

```diff
@@ -0,0 +1,29 @@
+# Glossary
+
+## Simulation Sequence Components
+
+**cache:** Dictionary of each stage and its vector of caches. Each cache tracks the last time status of a device changed in a simulation.
```

Comment:
Don't mention the type of object, just the definition for instance in this case: 

`Cache is the storage of the solution of a variable from a stage to be used by other stages in the simulation`